[2023-09-01 00:31:12,618] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2023-09-01 00:35:17,489] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2023-09-01 00:35:18,396] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2023-09-01 00:35:20,583] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2023-09-01 00:35:20,583] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2023-09-01 00:35:23,051] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2023-09-01 00:35:23,051] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2023-09-01 00:35:27,047] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2023-09-01 00:35:28,130] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2023-09-01 00:36:36,422] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2023-09-01 00:36:36,422] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2023-09-01 00:37:00,604] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2023-09-01 00:37:00,604] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2023-09-01 00:37:19,460] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2023-09-01 00:37:20,578] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2023-09-01 00:47:47,543] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2023-09-01 00:47:48,683] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2023-09-01 00:58:02,709] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2023-09-01 00:58:03,448] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2023-09-01 00:58:18,714] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
[2023-09-01 01:44:40,244] 	INFO 	23 	id_data_v2.py 	<module>() 	ID Data Started
[2023-09-01 01:49:03,427] 	INFO 	120 	id_data_v2.py 	<module>() 	ID Data Ended
[2023-09-01 01:49:04,306] 	INFO 	26 	day_data.py 	<module>() 	Day Data Started
[2023-09-01 01:49:06,942] 	INFO 	72 	day_data.py 	<module>() 	Day Data Ended
[2023-09-01 01:49:06,942] 	INFO 	78 	day_data.py 	<module>() 	Day Data Fusion Started
[2023-09-01 01:49:09,552] 	INFO 	124 	day_data.py 	<module>() 	Day Data Fusion Ended
[2023-09-01 01:49:09,552] 	INFO 	131 	day_data.py 	<module>() 	Day Data Combined Started
[2023-09-01 01:49:12,816] 	INFO 	161 	day_data.py 	<module>() 	Day Data Combined Ended
[2023-09-01 01:49:14,035] 	INFO 	37 	pair_data.py 	<module>() 	Pair Data Started
[2023-09-01 01:50:31,409] 	INFO 	111 	pair_data.py 	<module>() 	Pair Data Ended
[2023-09-01 01:50:31,409] 	INFO 	127 	pair_data.py 	<module>() 	Pair Data Fusion Started
[2023-09-01 01:50:58,364] 	INFO 	194 	pair_data.py 	<module>() 	Pair Data Fusion Ended
[2023-09-01 01:50:58,364] 	INFO 	201 	pair_data.py 	<module>() 	Pair Data Combined Started
[2023-09-01 01:51:13,281] 	INFO 	231 	pair_data.py 	<module>() 	Pair Data Combined Ended
[2023-09-01 01:51:14,516] 	INFO 	38 	bribe_data.py 	<module>() 	Bribe Data Started
[2023-09-01 02:02:10,154] 	INFO 	146 	bribe_data.py 	<module>() 	Bribe Data Ended
[2023-09-01 02:02:11,456] 	INFO 	38 	fee_data.py 	<module>() 	Fee Data Started
[2023-09-01 02:12:56,243] 	ERROR 	137 	fee_data.py 	<module>() 	Error occurred during Fee Data process. Error: 'address'
[2023-09-01 02:12:57,123] 	INFO 	24 	revenue_data.py 	<module>() 	Revenue Data Started
[2023-09-01 02:13:19,340] 	INFO 	72 	revenue_data.py 	<module>() 	Revenue Data Ended
